Workshop Quotes

SondraE opened the pull request:
## Related Issue(s)
[#265](https://github.com/oddbird/oddFlock/issues/265)


## Steps to test/reproduce
The quotes on the CSS Layouts Workshop page: /workshops/cascading-layouts/
The quotes on the Talks List page: /talks/

SondraE commented:
@mirisuzanne @jgerigmeyer Quotes listed in Hosted Workshop | Contact people who will promote
[#265](https://github.com/oddbird/oddFlock/issues/265) issue have been added to the workshop page and a couple to the main Talks page. Ready for review.

mirisuzanne reviewed (commented):
Do we want to break these up at all, or keep them at the same place on the page?

I believe Geoff is the editor at CSS Tricks. Currently we list title and venue as though it's the person's title _at that venue_ – but I think here we're combining career-generic titles, and links to people's personal sites? I don't think either title or venue is required, but I'm not sure the best solution here. Just seems a bit inconsistent maybe?

jgerigmeyer commented on src/images/quotes/ethan-marcotte.webp:
Is this file used?

jgerigmeyer reviewed (approved):
I have the same questions as Mia -- otherwise LGTM.

SondraE commented:
@jgerigmeyer @mirisuzanne 

> Break quotes up at all?

Broken up.

> I don't think either title or venue is required, but I'm not sure the best solution here.

Removed titles. 

> Image needed

No. Removed.
